content/docs/engine-dev/tools/project-manager/_index.md

```diff
@@ -1,167 +1,15 @@
-    class node3_sub1 PlainTextNode
-    class b3 LinkTextNode
-    click b3 "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp#L17" _blank
-    class c3 LinkTextNode
-    click c3 "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp#L31-L47" _blank
-    class d3 LinkTextNode
-    click d3 "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp#L49-L60" _blank
-
-    class node4 TextNode
-    click a4 "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreensCtrl.h" _blank
-    style a4 fill:#f9f,stroke:#76c4e2,stroke-width:4px
-    class node4_sub1 PlainTextNode
-    class b4 PlainTextNode
-
-    class node5 TextNode
-    click a5 "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenFactory.cpp" _blank
-    class node5_sub1 PlainTextNode
-    class b5 LinkTextNode
-    click b5 "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenDefs.h#L16-L33" _blank
-
-    class ProjectManager PlainTextNode
-    
-```
-
-## ScreenWidget
-
-```mermaid
-graph LR
-    classDef PlainTextNode fill:#0000, stroke:#0000, stroke-width:0px, text:#0000
-    classDef LinkTextNode fill:#0000, stroke:#2998c4, stroke-width:2px
-    classDef TextNode fill:#1111, stroke:#0000, stroke-width:0px, text:#0000
-    
-    subgraph _ [ ]
-    direction LR
-        subgraph __ [ ]
-            A(ScreenWidget)
-            B(Parent class to all non-trivial screens in ProjectManager)
-            C(Outlines all default transition functions)
-            D(Outlines all default screen refresh functions)
-            E(ScreensCtrl is implemented using ScreenWidget, thus transition logic is polymorphic)
-        end
-    end
-
-    C---D---B---E;
-    linkStyle 0 stroke:#0000,stroke-width:0px;
-    linkStyle 1 stroke:#0000,stroke-width:0px;
-    linkStyle 2 stroke:#0000,stroke-width:0px;
-
-    style A stroke:#f66,stroke-width:2px,stroke-dasharray: 5 5
-    click A "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenWidget.h" _blank
-    class _ PlainTextNode
-    class __ TextNode
-    class B PlainTextNode
-    class C LinkTextNode
-    click C "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenWidget.h#L66-L68" _blank
-    class D LinkTextNode
-    click D "https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenWidget.h#L60-L63" _blank
-    class E PlainTextNode
-
-
-```
\ No newline at end of file
+| Name | Description |
+|-|-|
+| [Bootup Sequence](./bootup-sequence.md) | Learn about how the Project Manager starts up. This will include all necessary steps in the sequence and identify some key classes that affect the rest of the Project Manager's architecture.|
```

Comment:
Not a link-breaker, but you don't need the `.md` in Markdown links. As a general rule we leave it off.
```suggestion
| [Bootup Sequence](./bootup-sequence) | Learn about how the Project Manager starts up. This will include all necessary steps in the sequence and identify some key classes that affect the rest of the Project Manager's architecture.|
```